CHANGELOG.md

```diff
@@ -47,6 +48,113 @@ Notable changes in reversed alphabetical order:
 
 The full list of included changes:
 
+- Backports ([#8624](https://github.com/paritytech/parity/pull/8624))
+  - Trace precompiled contracts when the transfer value is not zero ([#8486](https://github.com/paritytech/parity/pull/8486))
+    - Trace precompiled contracts when the transfer value is not zero
+    - Add tests for precompiled CALL tracing
+    - Use byzantium test machine for the new test
+    - Add notes in comments on why we don't trace all precompileds
```

Comment:
I don't understand this sentence, can you rephrase? I guess it should be something like `we don't trace all precompiled contracts` ?!

Reply:
It's a commit message. I am still undecided if we should keep all of them in the changelogs.